tomcat/versions-url

```diff
@@ -0,0 +1 @@
+https://github.com/apache/tomcat/tags
```

Comment:
nice catch. For `ide-urls` you can then use the `GithubBasedCrawler` to get the versions.